src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -278,6 +285,24 @@ internal void CreateExitModal(ExitGuide exitGuide)
             exitGuideWindow.IsOpen = true;
         }
 
+        /// <summary>
+        /// Shows/Hides the Popup based in if the DynamoView is Active or not
+        /// </summary>
+        /// <param name="isActive"></param>
+        internal void ManagePopupActivation(bool isActive)
+        {
+            if (GuideFlowEvents.IsAnyGuideActive)
+            {
+                if (currentGuide.CurrentStep.stepUIPopup.IsOpen == !isActive)
+                    currentGuide.CurrentStep.stepUIPopup.IsOpen = isActive;
+            }
+            else if (GuideFlowEvents.IsAnyGuideActive == false && GuideFlowEvents.IsGuideClosed == false)
+            {
+                if(exitGuideWindow != null && currentGuide.Name.Equals("Packages"))
```

Comment:
I remember we have a resource string for this

Reply:
@QilongTang changed to use a static string already defined.
https://github.com/DynamoDS/Dynamo/pull/13002/commits/8fbd9ac54e7491e9e36e35be28098bc93d5b8b24